cypress.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
❓ Do we need this file `cypress.json`?

Reply:
Not like this (won't have any effect). Right now we are setting the config file in `integration.sh` to be `./cypress/config/development.json`. 
Later (with diff envs, not only development) we can configure to pull the correct config file according to the env.